### ISSUE TITLE
error installing pypyodbc visual studio

### ISSUE THREAD

**Issue by MakabongweNkabinde** [Microsoft/PTVS#2897]
Hi, I'm trying to connet to a database using a python class and I read somewhere that I need pypyodbc. I tried installing that but I get an error:
 "[Window Title]
Visual Studio - Python support

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[V] Show details  [Close]

[Expanded Information]
```
Build: 3.0.17114.1

System.InvalidOperationException: Operation cannot be started because the environment is not configured.
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<AbortIfNotReady>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<InstallAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtensionProvider.<InstallPackage>d__38.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtension.<InstallPackage_Executed>d__14.MoveNext()
```"

**Comment by zooba** [comment]
Thanks for reporting this!

This particular issue has already been fixed in the next update of Visual Studio 2017. Make sure you install it when you get prompted and then you shouldn't see this again.

**Comment by zooba** [comment]
Dup of #2495, fixed in #2502
